notebooks/209-handwritten-ocr/209-handwritten-ocr.ipynb

```diff
@@ -0,0 +1,445 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0293f65e",
+   "metadata": {},
+   "source": [
+    "# Handwritten Chinese and Japanese OCR\n",
+    "\n",
+    "In this tutorial, we perform optical character recognition for handwritten Chinese (simplified) and Japanese. An OCR tutorial using the Latin alphabet is available in [notebook 208](../208-optical-character-recognition). This model is capable of processing only one line of symbols at a time. \n",
+    "\n",
+    "The models used in this notebooks are [handwritten-japanese-recognition](https://docs.openvinotoolkit.org/latest/omz_models_model_handwritten_japanese_recognition_0001.html) and [handwritten-simplified-chinese](https://docs.openvinotoolkit.org/latest/omz_models_model_handwritten_simplified_chinese_recognition_0001.html). To decode model outputs as readable text [kondate_nakayosi](https://github.com/openvinotoolkit/open_model_zoo/blob/master/data/dataset_classes/kondate_nakayosi.txt) and [scut_ept](https://github.com/openvinotoolkit/open_model_zoo/blob/master/data/dataset_classes/scut_ept.txt) charlists are used. Both models are available on [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fda2e1e0",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e0a6a0d5",
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import cv2\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from collections import namedtuple\n",
+    "from itertools import groupby\n",
+    "from openvino.inference_engine import IECore\n",
+    "from os import path"
```

Comment:
```suggestion
    "from pathlib import Path"
```
I changed the calls that use this too. I would like to standardize on using pathlib.